src/Waypoint/WaypointReaderSeeYou.cpp

```diff
@@ -299,6 +303,17 @@ WaypointReaderSeeYou::ParseLine(const TCHAR* line, Waypoints &waypoints)
     new_waypoint.comment = params[iDescription];
   }
 
+  if (iUserData < n_params) {
+    new_waypoint.details = params[iUserData];
+  }
+
+  if (iPics < n_params) {
+    for (tstring_view i : TIterableSplitString(params[iPics], _T(';'))) {
+      tstring p = _T("pics/");
```

Comment:
What's ... this?

Reply:
The problem is that params[iPics] comes as wide when on WIN64, and normal on UNIX. 
So this is the platform-neutral implementation, by using the tstring_view alias and _T for translation of the strings. 
The "pics/" directory is what is required by the CUP standard. As I didn't want to break files.embed for the waypoints_details.txt usecase i decided to add it here in the cup parser.

Reply:
Not the CUP standard, but the CUPX standard, and the CUP parser should not hard-code assumptions from another standard, or else something else will break.

Reply:
https://downloads.naviter.com/docs/SeeYou_CUP_file_format.pdf

Its not clear to me here where cup standard ends and cupx begins. 
The field is described in the cup standard document:

"A string field containing the filename(s) of an JPG image stored in pics/ folder of a seperate pics.zip file.See SeeYou CUPX file format specification. Multiple filenames must be separated by commas and string mustbe embraced in double quotes."

Since we don't want to implement CUPx's insane zip concatenation:
 * we would have a single archive with  points.cup and pics/foo.jpg
 * search: the archive and the XCSoarData for pics/foo.jpg

Reply:
Since this is an extension of the standard, with the column counter check, this shouldn't affect the parsing of the "legacy" cup files.

Reply:
Naviter didn't know the boundary between CUP and CUPX, so they mixed it up in the documentation. This is obviously a documentation mistake. They were not careful enough to document the difference.

Reply:
It looks like there isn't really a boundary. 
When i download cup files from soaringspot.com, all 14 fields are there. (although empty)
https://www.soaringspot.com/en_gb/nacional-clase-club-santa-rosa-2022/downloads

So the question is here, whether we try to read pictures when we have only the file and not an archive (cupx or otherwise.)

Reply:
The CUP parser doesn't know (and shouldn't be aware) if it reads a CUP file from a CUPX/ZIP container.